var/spack/repos/builtin/packages/symengine/package.py

```diff
@@ -35,7 +36,9 @@ class Symengine(Package):
     version('0.1.0', '41ad7daed61fc5a77c285eb6c7303425')
     version('develop', git='https://github.com/symengine/symengine.git')
 
-    variant('flint',        default=True,
+    variant('boost',        default=True,
```

Comment:
Ah, I see what the problem is! Thanks, I should have checked the logs more carefully.